src/components/app/details/cicdHistory/Artifacts.tsx

```diff
@@ -166,13 +168,41 @@ const CIProgressView = (): JSX.Element => {
     )
 }
 
-const CIListItem = ({ type, children }: CIListItemType) => {
+export const CIListItem = ({ type, userApprovalMetadata, triggeredBy, children }: CIListItemType) => {
     return (
         <div className={`mb-16 ci-artifact ci-artifact--${type}`} data-testid="hover-on-report-artifact">
-            <div className="bcn-1 flex br-4">
-                <img src={type === 'artifact' ? docker : folder} className="icon-dim-24" />
-            </div>
-            {children}
+            {type === 'approved-artifact' ? (
```

Comment:
Same here

Reply:
Done. I did extract the ApprovalInfoTippy but have moved the whole artifact part